packages/manager/apps/cloud/client/app/dbaas/logs/detail/inputs/logs-inputs.service.js

```diff
@@ -628,15 +641,19 @@ class LogsInputsService {
     return details;
   }
 
-  static transformInputToSave(input) {
+  transformInputToSave(input) {
+    if (trim(input.info.allowedNetworks) !== '') {
```

Comment:
```suggestion
    import isEmptry from "lodash/isEmpty";
    
    ...
    
    if (isEmpty(trim(input.info.allowedNetworks))) {
```
Is more expressive than `!== ''` :)

Reply:
Not valid, since isEmpty only handles Arrays and Objects. Here we check a String.

Reply:
You are absolutely right, but `string` are `array-like` 🙂 

In `JavaScript`, `array-like` means "has a `.length` property and indexed elements", for example, both those are valid:
- `"hello".length`
- `"hello"[0]`

This way, you can use `_.isEmpty` on a `string`!